test/acceptor_pool_SUITE.erl

```diff
@@ -332,7 +332,7 @@ shutdown_children(Config) ->
     [{_, Pid1, _, _}, {_, Pid2, _, _}] = acceptor_pool:which_children(Pool),
 
     {links, Links} = process_info(Pool, links),
-    [Acceptor] = Links -- [Pid1, Pid2, self()],
+    [Acceptor, _LSock] = Links -- [Pid1, Pid2, self()],
```

Comment:
Lets fetch `LSock` (`?config(socket, Config)`) and add it the substraction to avoid implementation detail ordering.